index.html

```diff
@@ -1106,8 +1106,8 @@ <h2 class="section-heading">FAQ</h2>
 										<div class="accordion-inner">
 											<p>
 
-												WearHacks is an international nonprofit organization whose mission is to make hardware education - with an emphasis on the domain of Wearable and Internet of Things technologies - accessible, approachable, and affordable to everyone.
-												Through our WearHacks hackathons, we give hackers access to cutting edge hardware devices with learning materials, create an environment whereby hackers feel comfortable exploring new domains, and reduce the price of experimentation and upfront investment from the hackers.
+												WearHacks is an international nonprofit organization whose mission is to make hardware education accessible, approachable, and affordable to everyone. With an initial focus on wearable technologies and the internet of things, WearHacks has expanded into emerging technologies such as augmented and virtual reality.
```

Comment:
* Internet of Things should be capitalized
* Don't know if we want to say initial focus and affordable